### DISCUSSION TITLE
Loading 3rd party JS libraries

### DISCUSSION THREAD

**Discussion by wtchen** [leptos-rs/leptos#1879] [Q&A]
I'd like to load d3 or Chart.js to build interactive charts. To do that, I would have to pull in those libraries somehow.

How would one do this? Seems like this is possible, so just want to see how people are doing this.

**Comment by gbj** [comment]
`wasm-bindgen` is the tool to integrate JavaScript libraries with Rust/WASM code.

See e.g., its docs on [importing functions from JS](https://rustwasm.github.io/docs/wasm-bindgen/examples/import-js.html).
